Course/Course/Presentation/Subviews/CustomDisclosureGroup.swift

```diff
@@ -95,7 +95,10 @@ struct CustomDisclosureGroup: View {
                                                 guard let chapterIndex = chapterIndex else { return }
                                                 guard let sequentialIndex else { return }
                                                 guard let courseVertical = sequential.childs.first else { return }
```

Comment:
Should we also display any toast error messages to the user in these possible cases?

Reply:
We will he handing gated content properly post MVP, so we will look into it as well.